gym_data.php

```diff
@@ -128,4 +267,6 @@
     unset($raid);
 }
 
+$p['token'] = checkForTokenReset();
```

Comment:
Consider renaming `checkForTokenReset` to make it clear that it returns something, e.g. `refreshCsrfToken`.